src/api.js

```diff
@@ -0,0 +1,13 @@
+export const API_ROOT = process.env.REACT_APP_API_ROOT
+export const API_KEY = process.env.REACT_APP_API_KEY
+
+export const params = {
+  'api-key': `${API_KEY}`
+}
+
+// NOTE: allows for additional parameters
+export const get = async(url) => {
+  url += '?'
+  Object.entries(params).map(([key, val]) => (url += `&${key}=${val}`))
```

Comment:
I think this will result in the query param starting with a question mark AND ampersand:
```
https://api.nytimes.com/svc/topstories/v2/science.json?&api-key=…
```

Reply:
You're right, it does, but that is acceptable in a URL query.